web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field.html.twig

```diff
@@ -46,14 +46,12 @@
 %}
 {% if label_hidden %}
   {% if multiple %}
-    <div{{ attributes.addClass("os2loop-title") }}>
-      {% for item in items %}
-        <div{{ item.attributes }}>{{ item.content }}</div>
-      {% endfor %}
-    </div>
+    {% for item in items %}
```

Comment:
I've wanted to make this change, but I didn't have the guts to do it – or the overview of the templates.

Reply:
I too am unsure if it will break something, but i think it needs to be done to have proper html for a default field

Reply:
We will have to look out for weird label styling